requirements.txt

```diff
@@ -1,9 +1,9 @@
 appdirs~=1.4
 cached-property~=1.4.3
 cli-helpers~=1.0.2
-git+https://gitlab.com/coala/coala-utils#egg=coala-utils-0.7.0
+coala-utils==0.7.0
 colorlog>=2.7,<4.0
-git+https://gitlab.com/jayvdb/package_manager#egg=dependency-management-0.5.0.dev0
+dependency-management
```

Comment:
Similar as before, use the git source.

Reply:
I'm not sure if that works, there is a issue regarding this by @jayvdb https://github.com/di/pytest-reqs/issues/25

Reply:
pytest-reqs fails to recognize it as an requirement.

Reply:
You can always adjust the pytest-reqs version.

Reply:
Well, I tried doing that, but it leads to some weird error not recognising pluggy and other 2-3 dependencies.